packages/contracts/src/2.0.0/protocol/AssetProxyOwner/AssetProxyOwner.sol

```diff
@@ -16,16 +16,14 @@
 
 */
 
-pragma solidity ^0.4.10;
+pragma solidity 0.4.10;
```

Comment:
Why the downgrade to `0.4.10`?

Reply:
The MultiSigWallet contracts have all been thoroughly audited with 0.4.10, so I'd rather not make any changes to their bytecode. Not really concerned about the efficiency of this contract since it will be used very infrequently.